src/redux/cofigureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import animeReducer from './anime/anime';
+
+const reducer = combineReducers({
+  animeReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(thunk, logger),
+);
```

Comment:
- [ ] Please do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data displayed on the browser console.